src/test/typescript-worker_test.ts

```diff
@@ -590,4 +590,74 @@ suite('typescript builder', () => {
     ];
     await checkTransform(files, expected, {}, cdn);
   });
+
+  /* TSX Compilation */
+  test('compiles tsx file to js', async () => {
```

Comment:
Similar test for jsx would be good. How about just adding a `.jsx` file to this same test.

Reply:
sounds good!

Reply:
done!